daemon/container.go

```diff
@@ -261,45 +263,45 @@ func validateContainerConfig(config *containertypes.Config) error {
 	return validateHealthCheck(config.Healthcheck)
 }
 
-func validateHostConfig(hostConfig *containertypes.HostConfig) error {
+func validateHostConfig(hostConfig *containertypes.HostConfig) (warnings []string, _ error) {
```

Comment:
Looks like we don't generate any warnings in `validateHostConfig` for now,  the returned `warnings` would be `nil`

Reply:
Correct but there might be one soon

Reply:
There is one now!